src/main/java/org/vandeseer/easytable/TableDrawer.java

```diff
@@ -139,11 +155,15 @@ public void draw(Supplier<PDDocument> documentSupplier, Supplier<PDPage> pageSup
     
     protected PDPage determinePageToDraw(int index, PDDocument document, Supplier<PDPage> pageSupplier) {
 		final PDPage pageToDrawOn;
-		if (index > 0 || document.getNumberOfPages() == 0) {
+		if ((index == 0 && startTableInNewPage) || index > 0 || document.getNumberOfPages() == 0) {
+			startTableInNewPage = false;
 			pageToDrawOn = pageSupplier.get();
 			document.addPage(pageToDrawOn);
 		} else 
-			pageToDrawOn = document.getPage(document.getNumberOfPages() - 1);	
+			pageToDrawOn = document.getPage(document.getNumberOfPages() - 1);
```

Comment:
Please add brackets around the bodies of `if' s and `else` s. This is a general recommendation. It's very easy to make mistakes on refactorings and it's pretty simple to prevent :wink:

Reply:
Will do.